cms/djangoapps/contentstore/views/item.py

```diff
@@ -173,6 +173,12 @@ def xblock_handler(request, usage_key_string=None):
             raise PermissionDenied()
 
         if request.method == 'GET':
+            # Populate user activity for tracking progress towards a user's course goals (for mobile app)
+            from lms.djangoapps.course_goals.models import UserActivity
```

Comment:
like below, import currently here because it was causing problems for cms tests - but will look into if i can avoid this